build/jobs/run-export-tests.yml

```diff
@@ -28,7 +28,8 @@ jobs:
             $environmentVariableName = $secret.Name.Replace("--","_")
 
             $secretValue = Get-AzKeyVaultSecret -VaultName $keyVault -Name $secret.Name
-            Write-Host "##vso[task.setvariable variable=$($environmentVariableName)]$($secretValue.SecretValueText)"
+            $plainValue = ([System.Net.NetworkCredential]::new("", $secretValue.SecretValue).Password).ToString()
```

Comment:
>$plainValue [](start = 12, length = 11)

I would add:
if ($plainValue) throw "Secret is empty" to catch this thing in future.